packages/functional-tests/lib/channels.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export enum FirefoxCommand {
+  FxAStatus = 'fxaccounts:fxa_status',
+  OAuthLogin = 'fxaccounts:oauth_login',
+  Logout = 'fxaccounts:logout',
```

Comment:
I tried using this in a couple other tests and ran into some type issues with `createCustomEventDetail` so it might need tweaking but I think this works for now.

We may want to move [these](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/lib/channels/firefox.ts#L1) from settings into a shared file.